.gitignore

```diff
@@ -97,6 +97,8 @@ data/benchmarks/ipad3/*
 data/benchmarks/ipad2/*
 data/benchmarks/iphone5/*
 data/benchmarks/maps/*
+data/cameras_to_ways.bin
+data/world_feed_integration_tests_data
```

Comment:
These files appear when tests are failed.
I'd prefer not to add them here, but rewrite tests (to use tmp dir if possible) later.

Reply:
No :)
These filies appear even when tests are passed.
Other such files and folders are already in gitignore
F.e:
```
data/gps_track.dat
# temporary files for downloader
data/settings.ini
data/bookmarks
```

Btw, data/world_feed_integration_tests_data will be automatically downloaded before test run, so it always will be in this folder and we need to add it to gitignore

Reply:
Let's leave it for the future corrections. It's not the scope of this PR

Reply:
I would prefer to have tests fixed instead of hiding their issues with gitignore. It increases chances to have concurrency issues.

Reply:
Ok, I removed it `data/cameras_to_ways.bin` from gitignore